lib/html-to-vdom.js

```diff
@@ -3,8 +3,23 @@ var parseHTML = require('./parse-html');
 
 module.exports = function initializeHtmlToVdom (VTree, VText) {
     var htmlparserToVdom = createConverter(VTree, VText);
-    return function convertHTML(html) {
+    return function convertHTML(html, options) {
```

Comment:
I'd say it would be more obvious to the reader of the code, if we actually put options as the first argument and then check if the second one is undefined.

```javascript
return function convertHtml (options, html) {
    var noOptions = _.isUndefined(html) && _.isString(options);
    var hasOptions = !noOptions;

    // was html supplied as the only argument?
    var htmlToConvert = noOptions ? options : html;
    var getVNodeKey = hasOptions ? options.getVNodeKey : undefined;

    var tags = parseHTML(htmlToConvert);

    var convertedHTML = htmlparserToVdom.convertTag(tags[0], getVNodeKey);

    return convertedHTML;
};
```